### ISSUE TITLE
parts - "Forelimb" and "hindlimb"

### ISSUE THREAD

**Issue by dustymc** [ArctosDB/arctos#7693]
"Forelimb" and "hindlimb" are in the code table--why use "leg" instead of "limb" to refer to something that might be one of those 2 things? This is confusing.

_Originally posted by @KatherineLAnderson in https://github.com/ArctosDB/arctos/issues/7670#issuecomment-2052467266_

**Comment by dustymc** [comment]
https://arctos.database.museum/info/ctDocumentation.cfm?table=ctpart_modifier#fore

https://arctos.database.museum/info/ctDocumentation.cfm?table=ctpart_modifier#hind

This is clearly denormalized and those parts (or modifiers, but they're much more versatile and probably doing other things) should be removed. 

(And I think there was some argument to keep '[leg](https://arctos.database.museum/info/ctDocumentation.cfm?table=ctspecimen_part_name#leg)' instead of directing that towards https://arctos.database.museum/info/ctDocumentation.cfm?table=ctspecimen_part_name#appendage, but I can't find it and not sure it ever made sense to me.)

**Comment by Jegelewicz** [comment]
Isn't this just a duplicate of #7670 ?

**Comment by dustymc** [comment]
I'd say the other way around - https://github.com/ArctosDB/arctos/issues/7670 has taken a long journey to become this and I'm clearly having trouble keeping up with it, maybe a more-focused issue would be useful to others as well. Or not, please organize however seems most useful!

**Comment by Jegelewicz** [comment]
> "Forelimb" and "hindlimb" are in the code table

After #7670 they won't be....
